cmd/invoke.go

```diff
@@ -62,9 +62,13 @@ dapr invoke --unix-domain-socket --app-id target --method sample --verb GET
 		client := standalone.NewClient()
 
 		// TODO(@daixiang0): add Windows support
-		if runtime.GOOS == "windows" && invokeSocket != "" {
-			print.FailureStatusEvent(os.Stderr, "unix-domain-socket option still does not support Windows!")
-			os.Exit(1)
+		if invokeSocket != "" {
+			if runtime.GOOS == "windows" {
+				print.FailureStatusEvent(os.Stderr, "The unix-domain-socket option is not supported on Windows")
+				os.Exit(1)
+			} else {
+				fmt.Println(print.WhiteBold("WARNING: This feature is currently only supported in preview mode"))
```

Comment:
Could we be explicit in the message which feature is in preview? There are lots of possible features that this warning could be referring to. Something like "Unix domain sockets are currently a preview feature". Also we have a `print.WarningStatusEvent()`